chains/evm/chain.go

```diff
@@ -48,7 +49,7 @@ func NewEVMChain(
 		blockInterval: blockInterval,
 		freshStart:    freshStart,
 		latestBlock:   latestBlock,
-		logger:        log.With().Str("domain", string(domainID)).Logger(),
+		logger:        log.With().Uint8("domainID", domainID).Logger(),
```

Comment:
I think we should decide either to use `domain` or `domainID` for this property in all loggers